boilerplates/sentry/files/pages/sentry/+Page.js

```diff
@@ -0,0 +1,21 @@
+/*{ @if (!(it.BATI.has("react") || it.BATI.has("vue") || it.BATI.has("solid"))) }*/
+export { Page };
+
+function Page() {
+  // language=HTML
+  return `
+<h1>Sentry Test Page</h1>
+<p id="sentry_state" style="color:red">
+    Sentry Client is not initialized! Vite Mode: ${import.meta.env.PROD ? "PROD" : "DEV"}
```

Comment:
Those `import.meta.env` are interpreted during Bati CLI execution, so here this probably always return `DEV`. Do not bother too much with pure HTML though, just make it as simple as possible.

Reply:
No, `import.meta.env` is different depending on how the code is run `pnpm dev` or `pnpm build && pnpm preview`. This page should help to find problems with the `Sentry Integration`.
The current setup does not include sentry in DEV and if you would try to test the integration it would fail.

Reply:
Oh, I though this was a `$+Page.js.ts` file, nevermind